src/lib.rs

```diff
@@ -749,6 +749,12 @@ impl<'a> ParsedMail<'a> {
             .unwrap_or_default();
         Ok(disposition)
     }
+
+    /// Returns the subparts, consuming the existing structure, allowing them to be
+    /// processed further independently.
+    pub fn into_subparts(self) -> Vec<ParsedMail<'a>> {
+        self.subparts
```

Comment:
You can just do this from the call site since the subparts field is public. What does this function buy you?

Reply:
You are right, I don't know why I didn't realize this earlier.